Makefile.kind

```diff
@@ -312,6 +312,41 @@ kind-install-cilium: check_deps kind-ready ## Install a local Cilium version int
 		--version= \
 		>/dev/null 2>&1 &
 
+GW_VERSION ?= $(shell grep "sigs.k8s.io/gateway-api" go.mod | awk -F'-' '{print (NF>1)?$$NF:$$0}')
```

Comment:
Can you update this command as per below? I recently fix one issue to handle v1.1.0-rc1 pattern.

https://github.com/cilium/cilium/pull/35189

Reply:
Thanks Tam, fixed!